test/kubernetes-test.sh

```diff
@@ -103,12 +103,9 @@ ${HOME}/go/bin/kubetest --provider=local --test \
 #   - TL;DR Cilium does not allow to specify pod CIDRs as part of the policy
 #     because it conflicts with the pod's security identity.
 #   - More info at https://github.com/cilium/cilium/issues/9209
-# - should enforce ingress policy allowing any port traffic to a server on a specific protocol
-#   - Cilium does not distinguish between UDP and TCP
-#   - More info at https://github.com/cilium/cilium/issues/9207
```

Comment:
It seems this comment was incorrectly mentioning #9207. The test was skipped first in https://github.com/cilium/cilium/commit/d846652fdb59b168914f7e57d889a1edaa0b00a0#diff-5b7396a53a4522682b55bc3b4bd7433ac49313b38e8398547dc2c66fdf9e2a3f (by @aanm) and the link was later added in https://github.com/cilium/cilium/blame/master/test/kubernetes-test.sh#L106 (by @joestringer). As far as I understand, both the sentence and the link are incorrect as we're here talking about not being able to match on e.g. TCP without a port, whereas the issue is about not being able to differentiate between TCP and UDP in service resolution.

Hence, it looks fine to just remove the comment.

Reply:
Yes that's a good point. I noticed that I've been mixing both issues with each other.

Reply:
:thinking: The test looks to me like it configures a TCP-only service then ensures that UDP connectivity fails:

https://github.com/kubernetes/kubernetes/blob/2b203348fbe7d05dec9e913e0fb50045a2474425/test/e2e/network/netpol/network_policy.go#L883-L899

Reply:
To me it looks like it's creating a TCP-only *policy*; I don't see any mention of services.